src/renderer/page/file/view.jsx

```diff
@@ -35,6 +36,16 @@ class FilePage extends React.PureComponent {
     }
   }
 
+  checkSubscriptionLatest(props) {
+    props.checkSubscriptionLatest(
+      {
+        channelName: props.claim.channel_name,
+        uri: buildURI({ contentName: props.claim.channel_name, claimId: props.claim.value.publisherSignature.certificateId }, false),
```

Comment:
@IGassmann this is just a formatting suggestion right? Can we have `Prettier` just do this rather than receive this message?

Reply:
Exactly. Prettier was previously run with every commit thanks to Husky. Since Husky is currently disabled due to a problem it was causing with the publishing the app, it doesn't automatically run. However, Prettier can still be run on all source files by entering the command `yarn format`. https://github.com/lbryio/lbry-app/blob/master/CONTRIBUTING.md#code-formatting

Reply:
@IGassmann the feedback from codacy seems largely useful so far, but the ones labelled `prettier/prettier` feel like noise. They're difficult to read and also all fixed in the same way - run `Prettier`.

Unless others disagree and assuming it's possible I'd disable just these for now.

Reply:
Done. Prettier errors are disabled until Husky come back.